attune/curve/_topas.py

```diff
@@ -209,8 +210,8 @@ def save(self, save_directory=None, plot=True, verbose=True, full=False):
                 _write_headers(new_crv, curve)
                 new_crv.write(f"{len(all_sibs)}\n")
                 for c in all_sibs:
+                    c = to_insert.pop(c.interaction, c)
```

Comment:
Prefer the version of the curve with the given interaction string from `to_insert`, if present